atest/robot/libdoc/doc_format.robot

```diff
@@ -44,12 +44,31 @@ Format in XML
     ROBOT    --docfor RoBoT   DocFormatHtml.py
     HTML     ${EMPTY}         DocFormatHtml.py
 
+Format in XML:HTML
+    [Template]    Test Format in XML:HTML
+    HTML    --format xMl:hTML            DocFormat.py
+    HTML    --docfor RoBoT -f XML:HTML   DocFormatHtml.py
+    HTML    -F ROBOT --format xml:html   DocFormat.py
+    
 Format from XML spec
     [Template]    NONE
     Run Libdoc    -F HTML ${TESTDATADIR}/DocFormat.py ${OUTXML}
     Copy File    ${OUTXML}    ${OUTPREFIX}-2.xml
     Test Format In XML    HTML    lib=${OUTPREFIX}-2.xml
 
+Format from XML:HTML spec
+    [Template]    NONE
+    Run Libdoc    -F ROBOT --format XML:HTML ${TESTDATADIR}/DocFormat.py ${OUTXML}
+    Copy File    ${OUTXML}    ${OUTPREFIX}-2.xml
+    Test Format In XML:HTML    HTML    lib=${OUTPREFIX}-2.xml
+
+Compare HTML from XML:HTML
+    [Template]    NONE
+    Run Libdoc    -F ROBOT --format XML:HTML ${TESTDATADIR}/DocFormat.py ${OUTPREFIX}_xml_with_html.xml
```

Comment:
I think you could use just `${OUTXML}` instead of `${OUTPREFIX}_xml_with_html.xml` here.

Reply:
choosen this name for better understandability. 
Reader should understand that this OUTXML contains HTML as doc.

Reply:
For me a generic and pre-existing name would feel a bit better but I don't feel strongly about it. We can leave it like this as well.